frontend/lazyLoadedView.js

```diff
@@ -0,0 +1,8 @@
+import Vue from 'vue'
+
```

Comment:
Maybe add a comment here explaining how we decide what files to put here?

Something like,

```
// NOTE: any modals opened with `LOAD_MODAL` go here
```